tdrs-frontend/src/mirage.js

```diff
@@ -0,0 +1,21 @@
+import { createServer } from 'miragejs'
+
+export default function startMirage(
+  { environment } = { environment: 'development' }
+) {
+  return createServer({
+    environment,
+
+    routes() {
+      this.namespace = 'mock_api'
+
+      this.post('/reports/', () => {
+        return 'Success'
+      })
+
+      // Allow unhandled requests to pass through
+      this.passthrough(`${process.env.REACT_APP_BACKEND_URL}/**`)
+      this.passthrough(`${process.env.REACT_APP_BACKEND_HOST}/**`)
+    },
+  })
+}
```

Comment:
This starts the mirage server with which we are stubbing the backend `/reports` endpoint to always succeed.

Reply:
As this endpoint is currently stubbed, the selected files will not be saved in the backend yet.